### DISCUSSION TITLE
Prevent bundle exe from immediately returning when running from command prompt in silent mode

### DISCUSSION THREAD

**Discussion by rwilm** [wixtoolset/issues#8679] [Questions] [answered]
When I run my bundle installer exe from the command prompt in silent mode, the exe returns almost immediately while the install/uninstall continues in the background. The command I am running is "MyBundle.exe /s". I am using the standard BA. How can I prevent the command from returning before the install/uninstall process has finished?

**Comment by robmen** [comment, marked as answer]
`start /wait MyBundle.exe /s`

See `start /?` for all the options. There are many. 

Note: This is a Windows behavior. It has nothing to do with installation.

**Reply by rwilm** [reply]
Awesome, thank you!
